Library/Homebrew/cask/tab.rb

```diff
@@ -0,0 +1,184 @@
+# typed: true
+# frozen_string_literal: true
+
+module Cask
+  class Tab
```

Comment:
Can't this be shared with the one for formulae?

Reply:
Probably yes... Originally I separated them because there were more significant differences and I thought it would make it more confusing if they were combined, but now that I'm looking at it they could probably be combined without too much complexity

Reply:
Yeh, I think it'd be nice to have a shared class (or method(s) which includes all the shared information for both, filename, etc. and then we have additional fields specific to only one in other classes/methods.

Generally I think we'll want to have all data that can be shared between both cases.